index.html

```diff
@@ -1,50 +1,53 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <link rel="shortcut icon" href="images/YT.png" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
-    <title>YouTube Website by Yayner</title>
-  </head>
-  <body>
-    <nav class="nav-flex">
-      <div class="nav-left nav-flex">
-        <img src="images/menu.png" class="menu" />
-        <img src="images/logo.png" class="logo" />
-      </div>
```

Comment:
Please use HTML 5 semantic tags. using div throughout your entire code may result in the unreadability of your code.

Reply:
Great job 👍 in mentioning to use proper HTML 5 semantic tags. But it's not totally clear which semantic tag to use in this particular case. You can elaborate and help by saying like `Kindly let's wrap the 'nav' tag inside a 'header' tag to follow HTML & CSS best practices.` You can say the same thing for the `div` tag by recommending the proper equivalent tag to replace the `div` with.